src/main/java/frc/utils/auto/PathPlannerUtils.java

```diff
@@ -0,0 +1,97 @@
+package frc.utils.auto;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.auto.NamedCommands;
+import com.pathplanner.lib.commands.PathfindingCommand;
+import com.pathplanner.lib.controllers.PPHolonomicDriveController;
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.path.PathPlannerPath;
+import com.pathplanner.lib.pathfinding.LocalADStar;
+import com.pathplanner.lib.pathfinding.Pathfinder;
+import com.pathplanner.lib.pathfinding.Pathfinding;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BooleanSupplier;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+public class PathPlannerUtils {
+
+	private static List<Pair<Translation2d, Translation2d>> dynamicObstacles = List.of();
+
+	public static void startPathfinder() {
+		setPathfinder(new LocalADStar());
+		scheduleWarmup();
+	}
+
+	private static void setPathfinder(Pathfinder pathfinder) {
+		Pathfinding.setPathfinder(pathfinder);
+	}
+
+	private static void scheduleWarmup() {
+		PathfindingCommand.warmupCommand().schedule();
+	}
+
+	public static void configurePathPlanner(
```

Comment:
```suggestion
	public static void configPathPlanner(
```
its a convention in swerve